GitUI/Editor/FileViewer.cs

```diff
@@ -607,14 +607,12 @@ public Task ViewFileAsync(string fileName, bool isSubmodule = false, Action? ope
             {
                 try
                 {
-                    var path = _fullPathResolver.Resolve(fileName);
-
-                    if (!File.Exists(path))
+                    if (!File.Exists(fullPath))
```

Comment:
This should be the way around - the path must be resolved via `_fullPathResolver`, i.e. resolved for the _current_ working directory, and not some arbitrary path.

Reply:
The full path is already resolved, no need to do it again